storage/innobase/include/srv0srv.h

```diff
@@ -1053,6 +1047,13 @@ bool
 srv_is_undo_tablespace(
 	ulint	space_id);
 
+bool
+srv_enable_redo_encryption_mk();
+bool
+srv_enable_redo_encryption_rk();
+bool
```

Comment:
New style for new functions (return type and function name on the same line)
Also while otherwise it could be possible to look the other way re. missing doc comments, now the relationship between three functions needs to be clarified.

Or maybe _mk _rk functions can be made statis srv0srv.cc?

Reply:
`_mk` and `_rk` is also used directly outside this function, I'll document them.